osbuild-composer.spec

```diff
@@ -331,27 +330,6 @@ fi
 # restart all the worker services
 %systemd_postun_with_restart "osbuild-worker@*.service" "osbuild-remote-worker@*.service"
 
-%package dnf-json
```

Comment:
Hmm, we should probably put `Obsoletes: osbuild-composer-dnf-json` into `osbuild-depsolve-dnf` so the package gets removed on upgrade, right?

Reply:
Created osbuild/osbuild#1524